### PR TITLE
Add appVersion key for heapster

### PR DESCRIPTION
**Pull request by AdamDang**
The key "appVersion" is needed for ci testing, it's missing in this yaml. That sometimes will cause testing failure.

### CODE REVIEW

File: stable/heapster/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 apiVersion: v1
 description: Heapster enables Container Cluster Monitoring and Performance Analysis.
 name: heapster
-version: 0.2.8
+version: 0.2.9
+appVersion: 1.3.0
+home: https://www.heapster.net/
```
Comment (timstoop) [inline stable/heapster/Chart.yaml]: I don't think this is the heapster you're looking for. The github link used in sources is the definitive home, I think.

**Review by timstoop — CHANGES_REQUESTED** [review]

File: stable/heapster/Chart.yaml (hunk 1 above)
Comment (AdamDang) [reply stable/heapster/Chart.yaml]: Thanks @timstoop ,Do you mean we should use the following URL? I've changed to it 
home: https://github.com/kubernetes/heapster

**Pull request merged**